.github/workflows/CI_merge.yml

```diff
@@ -29,8 +29,8 @@ jobs:
       - name: Lint
         run: npm run lint:eslint:all
 
-      - name: Unit Test
-        run: npm run test:unit
+      - name: Tests
```

Comment:
not a blocker, just a nitpick, but shouldn't it be consistent with CI_action_publish_master.yml and named "Test"?